src/sass/app/statement.scss

```diff
@@ -0,0 +1,9 @@
+#statement-table {
```

Comment:
we are using this identifier in `table.scss`, we could move this to the same place